src/pydot/core.py

```diff
@@ -1784,25 +1781,15 @@ def create(self, prog=None, format="ps", encoding=None):
         os.unlink(tmp_name)
 
         if process.returncode != 0:
-            message = (
-                '"{prog}" with args {arguments} returned code: {code}\n\n'
-                "stdout, stderr:\n {out}\n{err}\n"
-            ).format(
-                prog=prog,
-                arguments=arguments,
-                code=process.returncode,
-                out=stdout_data,
-                err=stderr_data,
+            code = process.returncode
+            print(
+                f'"{prog}" with args {arguments} returned code: {code}\n\n'
+                f"stdout, stderr:\n {stdout_data}\n{stderr_data}\n"
             )
-            print(message)
```

Comment:
Sections like this really display the benefits `f`-strings have over previous methods. 10 lines of code become 4.